mingw_files/nrnmingwenv.sh

```diff
@@ -85,6 +85,8 @@ copy() {
 copy mingw64/bin '
 as.exe
 ld.exe
+zlib1.dll
+libzstd.dll
```

Comment:
We should make sure that the changes from pull request #563 where we now automatically determinge needed dlls take precedence over this.

Reply:
Simply removed those, since `cp_dlls` is doing the job.